chiselFrontend/src/main/scala/chisel3/core/Data.scala

```diff
@@ -323,9 +382,7 @@ sealed class Clock extends Element(Width(1)) {
   def toPrintable: Printable = PString("CLOCK")
 
   override def do_asUInt(implicit sourceInfo: SourceInfo): UInt = pushOp(DefPrim(sourceInfo, UInt(this.width), AsUIntOp, ref))
-  override def do_fromBits(that: Bits)(implicit sourceInfo: SourceInfo, compileOptions: CompileOptions): this.type = {
-    val ret = Wire(this.cloneType)
-    ret := that
-    ret.asInstanceOf[this.type]
+  private[core] override def connectFromBits(that: Bits)(implicit sourceInfo: SourceInfo, compileOptions: CompileOptions) {
```

Comment:
This is same comment as the previous one about procedure syntax, I think there are a few of these in this PR